oojs.js

```diff
@@ -4,9 +4,31 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+  class Playlist{
+    constructor(title,songs){
+      this.title = title
+      this.songs = songs
+      this.favorites = 0
+    }
+    addSong(newSong){
+      this.songs.push('newSong')
```

Comment:
Yes, but here you're literally passing in the string `'newSong'` into the array every time. Remove the quotes so that it actually pushes the value of the argument.